app/src/main/java/com/ayush/anonytweet/DashBoard.java

```diff
@@ -273,7 +283,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
         // Handle action bar item clicks here. The action bar will
         // automatically handle clicks on the Home/Up button, so long
         // as you specify a parent activity in AndroidManifest.xml.
-        int id = item.getItemId();
+        // getId() is an internal android function.
```

Comment:
You don't need this confusing comment any more.

Reply:
Fixed.